octobot_trading/exchanges/exchange_manager.py

```diff
@@ -16,12 +16,12 @@
 import uuid
 
 from octobot_channels.util.channel_creator import create_all_subclasses_channel
-
 from octobot_commons.config_util import has_invalid_default_config_value
 from octobot_commons.constants import CONFIG_ENABLED_OPTION
 from octobot_commons.enums import PriceIndexes
 from octobot_commons.logging.logging_util import get_logger
 from octobot_commons.timestamp_util import is_valid_timestamp
+
```

Comment:
Why this newline ?

Reply:
Agree, not required, made by pycharm syntax refactor.